bin/run.py

```diff
@@ -401,15 +401,19 @@ def _get_expected_verdicts(self):
                 # Skip files where the key does not occur.
                 pass
 
-        subdir = self.short_path.parts[0].upper()
-        if subdir in config.VERDICTS:
-            if len(verdicts) > 0 and subdir not in verdicts:
-                error(
-                    f'Submission {self.short_path} must have implicit verdict {subdir} listed in @EXPECTED_RESULTS@.'
-                )
-                verdicts = [subdir] + verdicts
-            elif len(verdicts) == 0:
+        if self.path.parts[-3] == 'submissions':
```

Comment:
`[-3]` is ugly, but it works and is fine :shrug:

Reply:
Yeap, I didn't know a better way to make this consistently go right :sweat_smile: Ugly but works, indeed :upside_down_face: